src/routes/index.ts

```diff
@@ -303,6 +305,26 @@ router.route(config.OVERSEAS_ENTITY_REVIEW_URL)
   .get(overseasEntityReview.get)
   .post(overseasEntityReview.post);
 
+router.route(config.UPDATE_BENEFICIAL_OWNER_TYPE_URL)
+  .all(
+    authentication,
+    navigation.hasEntityUpdateDetails
+  )
+  .get(updateBeneficialOwnerType.get)
+  .post(...validator.beneficialOwnersType, checkValidations, updateBeneficialOwnerType.post);
+
+router.post(config.UPDATE_BENEFICIAL_OWNER_TYPE_SUBMIT_URL, authentication, navigation.hasEntityUpdateDetails, updateBeneficialOwnerType.postSubmit);
+
+router.route(config.UPDATE_MANAGING_OFFICER_TYPE_URL)
+  .all(
+    authentication,
+    navigation.hasEntityUpdateDetails
+  )
+  .get(updateManagingOfficerType.get)
+  .post(...validator.beneficialOwnersType, checkValidations, updateManagingOfficerType.post);
+
+router.post(config.UPDATE_MANAGING_OFFICER_TYPE_SUBMIT_URL, authentication, navigation.hasEntityUpdateDetails, updateManagingOfficerType.postSubmit);
```

Comment:
Need nav check to see we came from BOs. I'm happy to put a comment in or remove the nav check fro now until we get the journey designed.

Reply:
Comment placed